src/quo2/components/settings/data_item/view.cljs

```diff
@@ -70,19 +72,21 @@
       {:size  size
        :blur? blur?
        :theme theme}]
-     [left-subtitle
-      {:theme               theme
-       :size                size
-       :description         description
-       :icon                icon
-       :icon-color          icon-color
-       :blur?               blur?
-       :subtitle            subtitle
-       :customization-color customization-color
-       :emoji               emoji}])])
+     (if custom-subtitle
```

Comment:
Wondering if the design team should be requested to update `data-item` component. Wdyt?

Reply:
Yes, it should at least have a note imo that this is possible

Reply:
I will check with design team

Reply:
It seems we need to handle the addresses automatically ourselves. We can update our main `text` component to detect addresses and apply the needed styling, but I think adding a custom prop for this component is a better option. I have added a note.

Reply:
cool, btw with that note I meant in the Figma files. I've noticed they have been doing this for a few components recently so it's great if the designers can include it there too 👌